R/conjunct_test_linter.R

```diff
@@ -32,6 +33,14 @@
 #'   linters = conjunct_test_linter(allow_named_stopifnot = FALSE)
 #' )
 #'
+#' lint(
+#'   text = "dplyr::filter(
```

Comment:
sorry for the toil -- would you mind putting this into an inline expression? it'll be more consistent with examples from other linters. Typically when examples run to >1 line we write the examples  little differently (search for writeLines()), but I don't think it's necessary in this case.

Reply:
Like this? https://github.com/r-lib/lintr/pull/2110/commits/88734f31e1e7cc081916d999ce40a047e2c04937